Add ability to get latest infected addon ID's through a GitHub stored file

HonestResolv3 opened the pull request:
I hard-coded the theoretical URL that would house the file on GitHub if the PR was merged so do ensure that if this gets merged you test accordingly and make sure to update the URL variable if necessary

HonestResolv3 committed:
Add infected ID's to a .txt that can be editable

HonestResolv3 committed:
Add ability to get latest infected ID's through GitHub

A local copy of the latest IDs seen on the program is also used in case the program is not able to get the ID's remotely

HonestResolv3 committed:
Make coded URL read-only

HonestResolv3 commented:
I plan on incorporating the 2nd part of #2 soon as well as optimizing the way the .GMA files are being read (I do have a solution for that which will be in the next PR) so it will use much less memory and take less time

HonestResolv3 committed:
Remove file writing for local content, replaced with splitting response

LGuerrero13 commented on gif/Form1.cs:
```diff
@@ -179,5 +191,80 @@ private void lstInfectedFiles_DoubleClick(object sender, EventArgs e)
+            {
+                lblScannedFile.Invoke(new MethodInvoker(delegate
+                {
+                    displayError($"There was an error trying to use the program: {genEx.Message}");
```
Shouldn't have the text displayed on the lblScannedFile since messages will overflow outside of the window. Instead use a MessageBox.Show or something equivalent.

HonestResolv3 replied:
I'll go correct that, I was thinking of originally changing those message boxes I originally added

LGuerrero13 commented on gif/Form1.cs:
```diff
@@ -179,5 +191,80 @@ private void lstInfectedFiles_DoubleClick(object sender, EventArgs e)
+            updateAddonList();
+        }
+
+        private void btnUpdateIDs_Click(object sender, EventArgs e)
```
Rename the text for the update button. I was initially confused because I was expecting it to let me manually update/add IDs to the list, but found out it retrieves using the HTTP request to retrieve them from the file. This could be confusing to users if they don't already know.

HonestResolv3 replied:
I'll go and update that as well

LGuerrero13 reviewed (changes requested):
Add changes and then should be good for approval.

HonestResolv3 committed:
Rename btnUpdateID, use MessageBoxes for errors

Also got rid of some unused exceptions (IOException, UnauthorizedAccess) since files are no longer being used

HonestResolv3 commented:
I have resolved them now, so do pull the latest changes and send any additional comments/concerns my way

LGuerrero13 commented:
Approved! 😄

HonestResolv3 commented:
Thank you! I did do testing before making this PR and pushing these additional changes as well

I'll focus now on the stuff I mentioned above in the 2nd comment
